apps/omg_watcher/lib/omg_watcher/poc_event_handler.ex

```diff
@@ -0,0 +1,25 @@
+defmodule OMG.Watcher.PocEventHandler do
+  use GenServer
+  alias OMG.Watcher.DB.EthEvent
+
+  def start_link(subscribe_to) do
+    GenServer.start_link(__MODULE__, subscribe_to, name: subscribe_to)
+  end
+
+  def init(subscribe_to) do
+    :ok = OMG.Bus.subscribe(Atom.to_string(subscribe_to), link: true)
+    {:ok, %{subscribe_to: subscribe_to}}
+  end
+
+  def handle_info(
+        {:internal_event_bus, :sync_height, data},
+        state
+      ) do
+    case state.subscribe_to do
```

Comment:
What do you think if I do this per module instead of branching here, e.g. `ProcessExitEventHandler` and `DepositEventHandler`?

Reply:
@unnawut I completely agree. This is a POC just to show what can be done. not necessarily how!